docs/testnet-and-devnet/Testnet-Guide.md

```diff
@@ -46,5 +48,5 @@ alephium.mining.miner-addresses = [
 "193maApeJWrz9GFwWCfa982ccLARVE9Y1WgKSJaUs7UAx",
 "16fZKYPCZJv2TP3FArA9FLUQceTS9U8xVnSjxFG9MBKyY"
 ]
-
+alephium.api.api-key-enabled = false
```

Comment:
@polarker I just want your feedback on this addition. When I started my testnet node, it said that "api key is required, please add alephium.api.api-key = ...". After I added it and restarted my node, I couldn't anymore start mining from Swagger, I was gettng the message "api key missing". So, I added the `alephium.api.api-key-enabled = false` and then everything worked fine. I thought I could update the docs so that others don't face the same problem as me.

Am I missing something, or is this ok? Thanks!

Reply:
The design right now is that:
1. by default `api-key-enabled` is true
2. however, if the network interface is `127.0.0.1`, `api-key` is not needed.

Did you change your network interface `alephium.api.network-interface = "127.0.0.1"`?

Reply:
I did not. Maybe we should include that in the wiki then?

Reply:
Yes, the interface is changed. Your line to disable API key is great. I will later clean the config for testnet.